### ISSUE TITLE
What to use for benchmark concentrations?

### ISSUE THREAD

**Issue by limnoliver** [USGS-R/GLRI_CEC_2016#24]
From what I can tell, each site-date-chemical concentration is divided by each endpoint available for that compound - e.g., a single site-date-chemical concentration may become 6 benchmark values, one for each AcuteFish, AcuteInvert, and so on. Then all six of these values becomes part of the threshold computation. However, not all benchmark endpoints have equal representation across chemicals. Seems like we should choose the most relevant benchmark and stick with that? Or choose one at a time? 

Acute fish and acute invert have the broadest representation, with 109 and 110 chemical benchmarks, respectively.

**Comment by srcorsi-USGS** [comment]
Yes, true on the 6 values per site-date-chemical concentration. All of these can then be compared against a threshold. Definitely true that some benchmark columns are populated more completely than others. You have called out a common issue when dealing with these types of data. Acute data is commonly more available than chronic. Fish and invert more common than plants. One of our objectives here is to screen for potential biological effects regardless of what data is available, while recognizing that there are limitations (holes) in some of the data sets. My guess is that, in many (all?) instances, the benchmark values for chronic effects will be lower concentrations than acute, so acute will underestimate effects. If we can avoid that as much as possible, that would be preferred.

Our real objective with the benchmark data is to compare results with the toxcast results to help determine what EAR levels are relevant. It will be a challenge to get at this regardless of how we do it, but somehow we will need to use all of the benchmark data. It will be worth exploring the different options.
